pkg/testgridanalysis/testidentification/ocp_variants.go

```diff
@@ -140,10 +107,16 @@ func (v openshiftVariants) IdentifyVariants(jobName string) []string {
 		}
 	}()
 
+	// If a job is in never-stable, it is excluded from other possible variant aggregations
```

Comment:
it seems like this is the main benefit from having a strong list of never-stable jobs:  they don't pollute our variant aggregate results.  Was that discussed?

@deads2k i'd expect you to have strong opinions here.

Reply:
> it seems like this is the main benefit from having a strong list of never-stable jobs: they don't pollute our variant aggregate results. Was that discussed?

That was the intention, but at the moment [84 of 125 jobs](https://sippy.ci.openshift.org/sippy-ng/jobs/4.9?filters=%7B%22items%22%3A%5B%7B%22id%22%3A3089%2C%22columnField%22%3A%22current_pass_percentage%22%2C%22operatorValue%22%3A%22%3C%3D%22%2C%22value%22%3A%2240%22%7D%5D%7D&sort=asc&sortField=net_improvement) are 40 percent or less. Adding them back into their respective variants isn't going to taint them very much.

I think there's a distinction between "this job is permafailing, TRT has done some initial investigation, we've got a bug and someone's really actually looking at it, let's exclude it from the variant reporting" and "let's sweep these under the rug"

Reply:
@bparees No feedback from @deads2k here, can you have another look?

What if we meet in the middle and only keep jobs in never-stable if it's been at least investigated and had a bug opened?

That cuts the list in half or so, instead of removing most.

Reply:
> What if we meet in the middle and only keep jobs in never-stable if it's been at least investigated and had a bug opened?

that sounds reasonable to me.